src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange?: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}: Props) => {
+  const lastPage = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={() => {
+            if (currentPage > 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
Using `href="#prev"` will cause the page to jump to the top when clicked. Consider preventing the default action in the click handler to avoid this behavior. For example, you can modify the onClick handler to accept the event and call `event.preventDefault()`.